src/Azure.Functions.Cli/StaticResources/ExtensionsProj.csproj.template

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>

```

Comment:
This will have unintended consequences for .net6 in-proc customers. Until we can make this dynamic based on the customer's TFM, we need to use the lowest common denominator: which is net6

```suggestion
    <TargetFramework>net6.0</TargetFramework>
```